src/services/tripsService.js

```diff
@@ -83,6 +86,29 @@ class tripHelper {
 
     return allTrips;
   }
+
+  /**
+   * @param {string} page
+   * @param {string} limit
+   * @returns {object} most traveled centres infos
+   */
+  static async getTrendingCentres(page, limit) {
+    const pagination = paginate(page, limit),
+      mostTraveledCentres = await Booking.findAll({
```

Comment:
is the table called Booking or Bookings?

Reply:
@blaise82 I wasn't the one to name Models, so I imported available ```Booking``` model